website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -51,6 +51,13 @@ The volume staging directory for new CSI plugin tasks will now be
 mounted to the task's `NOMAD_TASK_DIR` instead of the
 `csi_plugin.mount_config`.
 
+#### Raft leadership transfer on error
+
+Starting with Nomad 1.3.0, when a Nomad leader fails to establish leadership it
+will attempted to gracefully transfer leadership to another eligible server in
```

Comment:
```suggestion
will attempt to gracefully transfer leadership to another eligible server in
```

Reply:
Thanks! Reading the note again I found it a bit confusing, so I end up re-writing  it 😄